Project.toml

```diff
@@ -10,6 +10,7 @@ LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MLStyle = "d8e11817-5142-5d16-987a-aa16d5891078"
 Multigraphs = "7ebac608-6c66-46e6-9856-b5f43e107bac"
 OMEinsum = "ebe7aa44-baf0-506c-a96f-8464559b3922"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
Commited by accident?

Reply:
Thanks! That makes sense. I will remove it. But it's a bit annoying since I load `Revise` automatically.

Reply:
put it in your global env